lib/pharos/host/ubuntu/scripts/configure-cri-o.sh

```diff
@@ -9,13 +11,51 @@ reload_daemon() {
     fi
 }
 
-mkdir -p /etc/systemd/system/crio.service.d
-cat <<EOF >/etc/systemd/system/crio.service.d/10-cgroup.conf
+export DEBIAN_FRONTEND=noninteractive 
+if dpkg -s cri-o-1.10 ; then
+    systemctl stop crio
+    systemctl disable crio
+    apt-get remove -y --purge cri-o-1.10
+    rm /etc/systemd/system/crio.service.d/10-cgroup.conf || true
+    systemctl daemon-reload
+fi
+apt-get install -y conntrack ibgpgme11 libseccomp2 libassuan
```

Comment:
```
 [Configure hosts @ 188.166.118.151] Pharos::SSH::RemoteCommand::ExecError: SSH exec failed with code 100: configure-cri-o.sh
+ set -e
+ . /usr/local/share/pharos/util.sh
+ export DEBIAN_FRONTEND=noninteractive
+ dpkg -s cri-o-1.10
dpkg-query: package 'cri-o-1.10' is not installed and no information is available
Use dpkg --info (= dpkg-deb --info) to examine archive files,
and dpkg --contents (= dpkg-deb --contents) to list their contents.
+ apt-get install -y conntrack ibgpgme11 libseccomp2 libassuan
Reading package lists...
Building dependency tree...
Reading state information...
E: Unable to locate package ibgpgme11
E: Unable to locate package libassuan
```

Reply:
Should probably be `libassuan0` `libgpgme11` - those seem to exist on both xenial and bionic.

Reply:
Fixed and tested on xenial & bionic.